fail-safely.md

```diff
@@ -0,0 +1,100 @@
+# Fail Safely
+
+Go doesn't have enums. (at least at the time of this writing and I have no idea if there are plans of adding them.)
+
+I've typically seen related values expressed as a set of constants.
+
+```go
+const (
+	Active  = "A"
+	Deleted = "D"
+	Pending = "P"
+)
+```
+
+or using iota. (no, stop it)
+
+```go
+const (
+  Active = iota + 1
+  Deleted
+  Pending
+)
+```
+
+However, in code, I may find something like:
```

Comment:
I think to better illustrate your idea here with the code sample, more of a specific example would help. Without knowing what the surrounding code is actually doing, it’s hard to know whether this is really a fail closed or fail open.

Reply:
I explain what's wrong with it _after_ the code example, which is probably where the issue is. I'll rework this to explain what to look for ahead of showing the code example.